roles/nginxplus/files/conf/http/templates/libwww-proxy-pass-staging.conf

```diff
@@ -3,9 +3,6 @@
     rewrite ^/research-data(.*)$ https://researchdata-staging.princeton.edu redirect;
     rewrite ^/AssignedSpaceApplication(.*)$ https://lockers-and-study-spaces-staging.princeton.edu/ redirect;
     rewrite ^/AssignedSpaces https://lockers-and-study-spaces-staging.princeton.edu/ redirect;
-    rewrite ^/research/databases https://libguides.princeton.edu/az.php redirect;
```

Comment:
@kevinreiss did you mean to remove them from staging?

Reply:
The deleted ones look like they redirected staging URLs to production libguides . . . I think? Do we have a staging site for libguides?

Reply:
Yes @carolyncole I removed them from staging intentionally. We are going to have a short period where Librarians can review the "old" entries in staging if needed in case something has been missed. There is not a staging site for libguides @acozine. We only have a single environment Springshare hosts for us.